packages/ui/components/elements/IngredientSearchInput/IngredientSearchInput.tsx

```diff
@@ -20,8 +20,8 @@ const IngredientSearchInput = () => {
   return (
     <Styled.SearchBoxContainer>
       <Styled.SearchBox>
-        <label htmlFor={id}><Styled.SearchIcons /></label>
-        <Styled.SearchInput id={id} type='text' className='input-search' placeholder='Type to Search...' />
+        <label htmlFor="input-search-field"><Styled.SearchIcons /></label>
+        <Styled.SearchInput id="input-search-field" type='text' className='input-search' placeholder='Type to Search...' />
```

Comment:
you broke Search input box, test it out.

now you se the is both working the same

Reply:
I have solved it